src/fcntl.rs

```diff
@@ -45,15 +46,28 @@ pub use self::posix_fadvise::{posix_fadvise, PosixFadviseAdvice};
 #[cfg(not(target_os = "redox"))]
 #[cfg(any(feature = "fs", feature = "process", feature = "user"))]
 libc_bitflags! {
+    /// Flags that control how the various *at syscalls behave.
     #[cfg_attr(docsrs, doc(cfg(any(feature = "fs", feature = "process"))))]
     pub struct AtFlags: c_int {
+        /// Used with [`unlink`](crate::unistd::unlink) to indicate that the path to remove is a
```

Comment:
It should be `unlinkat()`:

```suggestion
        /// Used with [`unlinkat`](crate::unistd::unlinkat) to indicate that the path to remove is a
```

And, our `unlinkat()` function does not take a `AtFlags` but a [`UnlinkatFlags`](https://github.com/nix-rust/nix/blob/bfcb2269d1f5e97723a1639cf6a33a85814afdf3/src/unistd.rs#L1312-L1317).

Reply:
In that case, this flag is totally unused.  I'll mark it as hidden and deprecated.